packages/cli/test/unit/commands/addons/create.unit.test.ts

```diff
@@ -79,6 +80,23 @@ describe('addons:create', function () {
       expect(stderr.output).to.contain('Creating heroku-postgresql:standard-0 on ⬢ myapp... ~$0.139/hour (max $100/month)\n')
       expect(stdout.output).to.equal('provision message\nCreated db3-swiftly-123 as DATABASE_URL\nUse heroku addons:docs heroku-db3 to view documentation\n')
     })
+    it('creates an add-on with proper output using old syntax with deprecation message', async function () {
+      await runCommand(Cmd, [
+        '--app',
+        'myapp',
+        '--as',
+        'mydb',
+        'heroku-postgresql:standard-0',
+        '--rollback',
+        '--follow',
+        'otherdb',
+        '--foo',
+      ])
+      expect(unwrap(stderr.output)).to.contain('Warning: You’re using a deprecated syntax with the [--rollback,--follow,--foo] flag')
+      expect(unwrap(stderr.output)).to.contain("Add a '--' (end of options) separator before the flags you’re passing through.")
+      expect(unwrap(stdout.output)).to.contain('For example: heroku addons:create -a myapp heroku-postgresql:standard-0 -- --rollback --follow otherdb --foo')
+      expect(unwrap(stdout.output)).to.contain('See https://devcenter.heroku.com/changelog-items/2925 for more info.')
```

Comment:
Here you can see both streams being used to output the full depreciation message. The actual output of the command is cosmetically correct, but the streams in which the deprecation warning is outputted is both `stderr` and `stdout`.